app/views/admin/settings/forms/_sponsors.html.erb

```diff
@@ -1,20 +0,0 @@
-<%= form_for(Settings::General.new,
```

Comment:
The headline was being used for the sponsorship widget in the sidebar which I've deleted.